llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -135,12 +136,20 @@ def __init__(
         self.init_device = init_device
         self.logit_scale = logit_scale
         self.no_bias = no_bias
-        self.verbose = verbose
         self.embedding_fraction = embedding_fraction
         self.norm_type = norm_type
         self.use_cache = use_cache
         self.init_config = init_config
         self.fc_type = fc_type
+        if verbose is not None:
+            warnings.warn(
+                DeprecationWarning(
+                    'verbose argument for MPTConfig will be deprecated in a future release. Use python_log_level instead.'
+                ))
+            logging.getLogger('llmfoundry').setLevel(
```

Comment:
what if we just do nothing if verbose is passed in? deprecationwarning can say that you should use proper pythong logging now. it won't break anyone's code if verbose just starts doing nothing.

Reply:
Done